server/prisma/datamodel.prisma

```diff
@@ -0,0 +1,67 @@
+enum TaskStatus {
+  COMPLETED
+  UNCOMPLETED
+}
+
+enum UserRole {
+  NEWBIE
+  BUDDY
+}
+
+type NewbieTask {
+  id: ID! @id
+  createdAt: DateTime! @createdAt
+  description: String! @default(value: "")
+  title: String! @default(value: "")
+  status: TaskStatus! @default(value: UNCOMPLETED)
+  newbie: Newbie!
+  notes: String
+}
+
+type BuddyTask {
+  id: ID! @id
+  createdAt: DateTime! @createdAt
+  description: String! @default(value: "")
+  title: String! @default(value: "")
+  status: TaskStatus! @default(value: UNCOMPLETED)
+  newbie: Newbie!
+}
+
+type Buddy {
+  id: ID! @id
+  name: String!
+  email: String! @unique
+  role: UserRole! @default(value: BUDDY)
+  password: String!
+  position: String
+  photo: String
+  phoneNumber: String
+  startDate: DateTime
+  notifications: Boolean! @default(value: false)
+  newbiesCount: Int! @default(value: 0)
+  newbies: [Newbie]!
+}
+
+type Newbie {
+  id: ID! @id
+  name: String!
+  email: String! @unique
+  role: UserRole! @default(value: NEWBIE)
+  password: String!
+  position: String
+  photo: String
+  phoneNumber: String
+  startDate: DateTime
+  notifications: Boolean! @default(value: false)
+  buddy: Buddy!
+  notes: String! @default(value: "")
+  tasksRating: Float! @default(value: 0)
+  buddyTasksRating: Float! @default(value: 0)
```

Comment:
What are these ones referring to?

Reply:
We gonna have diferent `Rating` for Buddy jorney and for Newbie. I mean when you are logged in as Newbie - you see rating only for newbie's completed taks. And for Buddy - it should be both newbie's completed taks plus byddy's tasks for current newbie

Reply:
`Progress`